processors/src/main/java/com/axellience/vuegwt/processors/component/template/parser/TemplateParser.java

```diff
@@ -75,10 +83,12 @@
      * @param messager Used to report errors in template during Annotation Processing
      * @return A {@link TemplateParserResult} containing the processed template and expressions
      */
-    public TemplateParserResult parseHtmlTemplate(String htmlTemplate,
-        TemplateParserContext context, Messager messager)
+    public TemplateParserResult parseHtmlTemplate(TypeElement componentTypeElement, String htmlTemplate,
```

Comment:
You can remove the `componentTypeElement` parameter as it is already in context: `context.getComponentTypeElement()`